lib/ProductOpener/Packaging.pm

```diff
@@ -505,6 +505,15 @@ sub apply_rules_to_augment_packaging_component_data ($product_ref, $packaging_re
 		$packaging_ref->{"shape"} = "en:coffee-capsule";
 	}
 
+	# If the shape is bottle and the material is glass, mark recycling as recycle
+	if (    (defined $packaging_ref->{"shape"})
+		and ($packaging_ref->{"shape"} eq "en:bottle")
+		and (defined $packaging_ref->{"material"})
+		and ($packaging_ref->{"material"} eq "en:glass"))
```

Comment:
`		and ($packaging_ref->{"material"} eq "en:glass"))`

It would be better to check if the material is en:glass or a child of en:glass.
You can use Tags.pm is_a() function for that:

is_a ($tagtype, $child, $parent)